### PR TITLE
Enable publishing E2E test data package.

### PR DESCRIPTION
**Pull request by dslzuha**
## Bug

Fixes: Link_to_issue  
Regression: Yes/No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: Details_about_the_fix  

## Testing/Validation

Tests Added: Yes/No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

**Review by dslzuha — COMMENTED** [review]
Needed approval

File: build/vsts_build.yaml [hunk 1]
```diff
@@ -212,6 +212,14 @@ phases:
       msbuildArguments: "/p:BuildNumber=$(BuildNumber) /p:IsVsixBuild=true"
     condition: " and(succeeded(), eq(variables['BuildRTM'], 'false'))"
 
+  - task: PowerShell@1
```
Comment (nkolev92) [inline build/vsts_build.yaml:215]: Why?

This is NuGet's internal CI configuration. 
We removed the e2e test data package generation on purpose. 

The idea is that if needed, the test data package will be updated manually. 

See https://github.com/NuGet/NuGet.Client/pull/2804.

**Comment by nkolev92** [comment]
Closing since there was no reply.

File: build/vsts_build.yaml (hunk 1 above)
Comment (dslzuha) [reply build/vsts_build.yaml:215]: thanks

**Comment by rrelyea** [comment]
You seem to be opening a number of PRs in our repo without significant changes. Perhaps you are learning to use GitHub and did not intend to send them to us. Please be aware that other people may be working in repos you do not own, so if you are testing, we suggest you create your own repo and follow steps similar to this: https://guides.github.com/activities/hello-world/

If you are hoping to contribute to nuget.client repro, please make sure to read https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md guidelines and start with step 1 (open an issue), not step 5 (open a PR).

**Pull request closed without merging**